src/reactionsystem.jl

```diff
@@ -205,7 +205,7 @@ end
 # Checks if a metadata input has an entry :only_use_rate => true
 function metadata_only_use_rate_check(metadata)
     only_use_rate_idx = findfirst(:only_use_rate == entry[1] for entry in metadata)
-    isnothing(only_use_rate_idx) && return true
+    isnothing(only_use_rate_idx) && return false
```

Comment:
This would have been caught by (and actually was) regular CI. The problem was that when we merged the noise scaling one (and the other early MTK v9 PRs) all tests was erroring, so we merged without CI. 

In this case I checked the new features manually, mainly using the newly written tests, and not the ones unrelated to the PR, hence it slipped through. This is just a very good example while we are right to not merge any code if not all of CI is passing.